packages/typespec-autorest/src/openapi.ts

```diff
@@ -848,7 +848,7 @@ function createOAPIEmitter(
     for (const data of response.responses) {
       if (data.headers && Object.keys(data.headers).length > 0) {
         openapiResponse.headers ??= {};
-        for (const [key, value] of Object.entries(data.headers)) {
+        for (const [key, value] of Object.entries(data.headers).sort()) {
```

Comment:
So this was actually done explicitly in the schema sorter to keep the order, I don't really remmber why https://github.com/Azure/typespec-azure/blob/dbe7d3afebba18d1ae9f49b2861a7873e033389e/packages/typespec-autorest/schema/autorest-openapi-schema.tsp#L306-L307


But if we wnat to fix this its the schema we should update not sort here.